plugins/flytekit-dask/flytekitplugins/dask/resources.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional
+
+import flytekit.models.task as _task_models
+from flytekit import Resources
+
+_ResouceName = _task_models.Resources.ResourceName
+_ResourceEntry = _task_models.Resources.ResourceEntry
+
+
+def _convert_resources_to_resouce_entries(resources: Resources) -> List[_ResourceEntry]:
```

Comment:
Seems like we can reuse this [method](https://github.com/flyteorg/flytekit/blob/c665616f12ae5fba072c4e9c37848f38cd14e913/flytekit/core/node.py#L120-L149)